main/FatesHistoryInterfaceMod.F90

```diff
@@ -4968,19 +5301,122 @@ subroutine define_history_vars(this, initialize_variables)
          upfreq=1, ivar=ivar, initialize=initialize_variables,                 &
          index = ih_fnrtc_si)
 
+    call this%set_history_var(vname='FATES_FROOTC_SL', units='kg m-3',                        &
+         long='Total carbon in live plant fine-roots over depth', use_default='active', &
+         avgflag='A', vtype=site_soil_r8, hlms='CLM:ALM', upfreq=1,         &
+         ivar=ivar, initialize=initialize_variables, index = ih_fnrtc_sl )
+    
     call this%set_history_var(vname='FATES_REPROC', units='kg m-2',            &
          long='total biomass in live plant reproductive tissues in kg carbon per m2', &
          use_default='active', avgflag='A', vtype=site_r8, hlms='CLM:ALM',     &
          upfreq=1, ivar=ivar, initialize=initialize_variables,                 &
          index = ih_reproc_si)
 
-    call this%set_history_var(vname='FATES_CEFFLUX', units='kg m-2 s-1',       &
-         long='carbon efflux, root to soil, in kg carbon per m2 per second',   &
-         use_default='active', avgflag='A', vtype=site_r8, hlms='CLM:ALM',     &
-         upfreq=1, ivar=ivar, initialize=initialize_variables,                 &
-         index = ih_cefflux_si)
 
+    ! Output specific to the chemical species dynamics used (parteh)
+    select case(hlm_parteh_mode)
+    case (prt_cnp_flex_allom_hyp)
+
+       call this%set_history_var(vname='FATES_L2FR', units='kg kg-1',                   &
+            long='The leaf to fineroot biomass multiplier for target allometry', & 
+            use_default='active', &
+            avgflag='A', vtype=site_r8, hlms='CLM:ALM', upfreq=1,    &
+            ivar=ivar, initialize=initialize_variables, index = ih_l2fr_si)
+       
+       call this%set_history_var(vname='FATES_RECL2FR_CANOPY_PF', units='kg kg-1',                   &
+            long='The leaf to fineroot biomass multiplier for recruits (canopy)', & 
+            use_default='active', &
+            avgflag='A', vtype=site_pft_r8, hlms='CLM:ALM', upfreq=1,    &
+            ivar=ivar, initialize=initialize_variables, index = ih_recl2fr_canopy_pf)
+
+       call this%set_history_var(vname='FATES_RECL2FR_USTORY_PF', units='kg kg-1',                   &
```

Comment:
I might had been worried that the var name was going to get too long and chopped by the host models. But I can at least try adding an underscore to see if it works, it would be nicer.